ethcore/src/block.rs

```diff
@@ -107,13 +108,17 @@ pub struct ExecutedBlock {
 	pub state: State<StateDB>,
 	/// Transaction traces.
 	pub traces: Tracing,
+	/// Writer for persisting storage changes.
+	pub storage_writer: Box<storage_writer::StorageWriter>,
```

Comment:
We should reuse `Tracing` for this instead of introducing additional entries.

Reply:
Cool, point taken. I had originally put this adjacent to `Tracing` because it felt similar but different - i.e. if a block can include knowledge of tracing, maybe it can also include knowledge of storage writing, but they shouldn't be combined if they're conceptually distinct. Happy to take a look at combining them if that seems like a better way to avoid introducing complexity.